flink-ml-lib/src/test/java/org/apache/flink/ml/feature/stringindexer/StringIndexerTest.java

```diff
@@ -53,6 +55,9 @@ public class StringIndexerTest extends AbstractTestBase {
     private Table trainTable;
     private Table predictTable;
 
+    private Table trainTableWithNull;
```

Comment:
Can you simplify the test by adding null values in `trainTable` and `predictTable`? In this case, the test case would be simpler.

Moreover, it would be great if we also update the python test.

Reply:
Thanks for your suggestion. I've updated the PR.  Please check the changes.